assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceTilesWidget/PlaceholderTile.js

```diff
@@ -0,0 +1,134 @@
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { useDispatch, useSelect } from 'googlesitekit-data';
+import { CORE_SITE } from '../../../../../../googlesitekit/datastore/site/constants';
+import { CORE_UI } from '../../../../../../googlesitekit/datastore/ui/constants';
+import { CORE_USER } from '../../../../../../googlesitekit/datastore/user/constants';
+import { MODULES_ANALYTICS_4 } from '../../../../datastore/constants';
+import Link from '../../../../../../components/Link';
+import NoAudienceBannerGraphic from '../../../../../../../svg/graphics/no-audience-banner-graphic.svg';
+import { AUDIENCE_SELECTION_PANEL_OPENED_KEY } from '../AudienceSelectionPanel/constants';
+
+export default function PlaceholderTile( { Widget } ) {
+	const hasConfigurableNonDefaultAudiences = useSelect( ( select ) => {
+		const configuredAudiences =
+			select( CORE_USER ).getConfiguredAudiences();
+
+		const configurableAudiences =
+			select( MODULES_ANALYTICS_4 ).getConfigurableAudiences();
+
+		return configurableAudiences.some(
+			( audience ) =>
+				audience.audienceType !== 'DEFAULT_AUDIENCE' &&
+				! configuredAudiences.includes( audience.name )
+		);
+	} );
+
+	const audienceLearnMoreURL = useSelect( ( select ) =>
+		select( CORE_SITE ).getGoogleSupportURL( {
+			path: '/analytics/answer/12799087',
+		} )
+	);
+
+	const { setValue } = useDispatch( CORE_UI );
+
+	const AnalyticsLink = (
+		<Link
+			secondary
+			href={ audienceLearnMoreURL }
+			external
+			hideExternalIndicator
+		/>
+	);
+
+	return (
+		<Widget className="googlesitekit-audience-segmentation-tile-placeholder">
+			<div className="googlesitekit-audience-segmentation-tile-placeholder__container">
+				<NoAudienceBannerGraphic className="googlesitekit-audience-segmentation-tile-placeholder__image" />
+				<div
+					className={ classnames(
+						'googlesitekit-audience-segmentation-tile-placeholder__body',
+						{
+							'googlesitekit-audience-segmentation-tile-placeholder__body--without-selectable-audiences':
+								! hasConfigurableNonDefaultAudiences,
+						}
+					) }
+				>
+					<h3 className="googlesitekit-audience-segmentation-tile-placeholder__title">
+						{ hasConfigurableNonDefaultAudiences
+							? __(
+									'Compare your group to other groups',
+									'google-site-kit'
+							  )
+							: __(
+									'Create more visitor groups',
+									'google-site-kit'
+							  ) }
+					</h3>
+					<p className="googlesitekit-audience-segmentation-tile-placeholder__description">
+						{ hasConfigurableNonDefaultAudiences
+							? createInterpolateElement(
+									__(
+										'<SelectGroupLink>Select</SelectGroupLink> another group to compare with your current group or learn more about how to group site visitors in <AnalyticsLink>Analytics</AnalyticsLink>',
```

Comment:
According to the [Figma design](https://www.figma.com/design/7pSrkEy8t00BcYRAi9LjjH/Audience-Segmentation?node-id=1046-40345&m=dev), the links `Select` and `Analytics` have a `font-weight: 500`.

![Screenshot 2024-08-16 at 3 53 34 PM](https://github.com/user-attachments/assets/39b88259-6225-4d03-bce0-9bf79512cd69)

Reply:
Thanks, good spot! This is fixed.